Altis_Life.Altis/core/actions/fn_pickupMoney.sqf

```diff
@@ -1,45 +1,42 @@
-#include "..\..\script_macros.hpp"
-/*
-    File: fn_pickupMoney.sqf
-    Author: Bryan "Tonic" Boardwine
-
-    Description:
-    Picks up money
-*/
-params [
-    ["_money", objNull, [objNull]]
-];
-
-if ((time - life_action_delay) < 1.5) exitWith {
-    hint localize "STR_NOTF_ActionDelay";
-    _money setVariable ["inUse",false,true];
-};
-if (isNull _money || {player distance _money > 3}) exitWith {
-    _money setVariable ["inUse",false,true];
-};
-
-private _value = (_money getVariable "item") select 1;
-if (!isNil "_value") exitWith {
-    deleteVehicle _money;
-
-    _value = switch (true) do {
-        case (_value > 20000000) : {100000}; //VAL>20mil->100k
-        case (_value > 5000000) : {250000}; //VAL>5mil->250k
-        default {};

```

Comment:
I had the idea to just relace `default {};` with `default {_value};` which did the trick, but your approach of a basically complete rewrite is way cleaner. Nice job.